charts/app-config/values-integration.yaml

```
@@ -1810,6 +1810,7 @@ govukApplications:
 
   - name: release
     helmValues:
+      appImage.arch: arm64
```

Does this definitely get transformed into an object `appImage` containing a key `arch`, rather than being passed through as just a single key named `appImage.arch` (which wouldn't come out as `.Values.appImage.arch` in the chart template)?

I dunno if Argo CD does something special here but with regular Helm values I'm pretty sure you'd have to write it as:
```suggestion
      appImage:
        arch: arm64
```

---

I did think about this just after I pushed it and thought "that might not work properly".